src/functions/process.go

```diff
@@ -0,0 +1,118 @@
+package functions
+
+import (
+	"Scruticode/src/constants"
+	"Scruticode/src/types"
+	"log"
+	"strings"
+)
+
+var actions = map[string]types.ActionFunc{
```

Comment:
Mentioned above

Reply:
Fixed this, since it was used only in one place I've moved the map inside the function that was being called from.